packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -235,6 +228,16 @@ watch(
 		} else if (openedAssetRoute.value && openedAssetRoute.value.assetName) {
 			tabStore.addTab(projectContext.value, openedAssetRoute.value);
 		}
+
+		const overviewResource = {
+			assetName: 'Overview',
+			pageType: ProjectPages.OVERVIEW,
+			assetId: ''
```

Comment:
I think this needs to be `underfined`

Reply:
@YohannParis unfortunately undefined causes mutliple 'Overview' tabs to be open when added with the addTabs function.  This worked with the router - however if they are being added through .addTabs it needs to be an empty string to passes the if on L249